docs/setup/README.md

```diff
@@ -0,0 +1,96 @@
+## Managed Anthos Service Mesh on GKE with Terraform 
```

Comment:
**Question:** To stick to the goal of having Tech Writers own all wording and instructions (and have DPEs focus on the raw code), do we plan on eventually moving these instructions into its own separate tutorial (or Quickstart) on cloud.google.com?

I guess we already have some Terraform tutorials on cloud.google.com, but they're "community" tutorials. [See left nav here](https://cloud.google.com/service-mesh/docs/unified-install/install-anthos-service-mesh-command
).
<img width="318" alt="Screen Shot 2022-05-02 at 3 13 20 PM" src="https://user-images.githubusercontent.com/10292865/166310278-e17ad151-4085-45d1-8910-6a8db3e41989.png">

Adding an associated tutorial/quickstart in cloud.google.com (later) will also let us preserve the truth of this sentence in `docsREADME.md`:
> This directory contains code samples for cloud.google.com.

Reply:
these TF tutorials are outdated, but I believe we will have this as a tutorial on how to setup MCP!